### ISSUE TITLE
Error with cache import

### ISSUE THREAD

**Issue by PencilNotPen** [dmunozv04/iSponsorBlockTV#2]
Traceback (most recent call last):
  File "/Users/test/Downloads/iSponsorBlockTV-main/main.py", line 5, in <module>
    from cache import AsyncTTL
ModuleNotFoundError: No module named 'cache'

Any ideas? I think dependencies are installed correctly from requirements.txt and API key is saved in .json file.

EDIT: requirements.txt still lists async_lru as dependency instead of async-ttl. I installed async-ttl manually with "pip install async-ttl", but I get the same error still. All other dependencies seem to be installed. No idea what the module cache error is. Thanks

**Comment by dmunozv04** [comment]
Hello, could you try installing the `async-cache` module? Did you clone the repo with git or did you just download a zip file?

**Comment by dmunozv04** [comment]
Can you confirm that after the PR from @HaltCatchFire it works?

**Comment by PencilNotPen** [comment]
Hi guys, thanks for the replies. I haven't been near my ATV recently to test again. 

I downloaded the .zip, didn't clone the repo with git.

I have now installed async-cache, and I think the error is resolved, although as I said, I am not with my ATV so I can only check by running main.py and seeing if it shows my original error or not, I can't actually see if the tool is working overall.

It now errors at:
